packages/wallet/src.ts/index.ts

```diff
@@ -240,6 +240,20 @@ export class Wallet extends Signer implements ExternallyOwnedAccount, TypedDataS
 		return Wallet.fromMnemonic(mnemonic, options.path, options.locale);
 	}
 
+	async createAccount(pubKey: BytesLike, initialBalance?: BigInt): Promise<TransactionResponse> {
+		if (!initialBalance) initialBalance = BigInt(0);
+		const signed = await this.signTransaction({
+			gasLimit: 10000,
```

Comment:
I don't think we need `gasLimit` in this tx

Reply:
If the gasLimit is not specified it throws this error:
```
      Error: cannot estimate gas; transaction requires manual gas limit (tx="[object Object]", code=UNPREDICTABLE_GAS_LIMIT, version=abstract-signer/5.5.0)
      reason: cannot estimate gas; transaction requires manual gas limit
      code: UNPREDICTABLE_GAS_LIMIT
      tx: [object Object]
```

Reply:
I guessed so. We have the option to modify the `throw` logic in order to recognize `AccountCreate` transactions. You can use the `isCreateAccount` flag or check if the `customData.publicKey` exists.

Can we say that if a `customData.publicKey` exists - then this is an `AccountCreate` transaction? 
If so - i think we can remove the `customData.isCreateAccount` flag.

Reply:
I removed `isCreateAccount`. Only `customData.publicKey` is used to determine if the tx is `createAccount`